<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Pagina01</class>
 <widget class="QWidget" name="Pagina01">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>834</width>
    <height>509</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelData">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QToolButton" name="prevMonth">
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QComboBox" name="mesi">
       <property name="minimumSize">
        <size>
         <width>1</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <item>
        <property name="text">
         <string>gennaio</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>febbraio</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>marzo</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>aprile</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>maggio</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>giugno</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>luglio</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>agosto</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>settembre</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ottobre</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>novembre</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>dicembre</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxAnno">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::PlusMinus</enum>
       </property>
       <property name="keyboardTracking">
        <bool>true</bool>
       </property>
       <property name="minimum">
        <number>2010</number>
       </property>
       <property name="maximum">
        <number>9999</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="nextMonth">
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCalendarWidget" name="calendarWidget">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <bold>false</bold>
       <stylestrategy>PreferDefault</stylestrategy>
       <hintingpreference>PreferNoHinting</hintingpreference>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="minimumDate">
      <date>
       <year>2010</year>
       <month>9</month>
       <day>14</day>
      </date>
     </property>
     <property name="gridVisible">
      <bool>true</bool>
     </property>
     <property name="horizontalHeaderFormat">
      <enum>QCalendarWidget::LongDayNames</enum>
     </property>
     <property name="navigationBarVisible">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>nextMonth</sender>
   <signal>clicked()</signal>
   <receiver>calendarWidget</receiver>
   <slot>showNextMonth()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>617</x>
     <y>382</y>
    </hint>
    <hint type="destinationlabel">
     <x>321</x>
     <y>184</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>prevMonth</sender>
   <signal>clicked()</signal>
   <receiver>calendarWidget</receiver>
   <slot>showPreviousMonth()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>25</x>
     <y>382</y>
    </hint>
    <hint type="destinationlabel">
     <x>321</x>
     <y>184</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
